nodejs/eks/nodegroup.ts

```diff
@@ -44,9 +44,16 @@ export interface Taint {
 export interface NodeGroupBaseOptions {
 
     /**
-     * The IDs of the explicit node subnets to attach to the worker node group.
+     * The explicit subnet IDs to use for the worker node group.
      *
-     * This option overrides clusterSubnetIds option.
+     * The IDs used must be a subset, or the whole set of IDs passed to the
```

Comment:
This section is confusing. It looks like this overrides default behavior, but the explanation seems overly complex. I'd focus on what it overrides.